test/Tester/StorageFacet/Feature.Abstractions/StorageFeatureAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using Orleans;
+
+namespace Tester.StorageFacet.Abstractions
+{
+    [AttributeUsage(AttributeTargets.Parameter)]
+    public class StorageFeatureAttribute : FacetAttribute, IStorageFeatureConfig
```

Comment:
I was using the term 'feature' to indicate what part of the code dealt with a set of functionality we wished to expose, and 'facet' to refer to the general facet system for which features are exposed.
I left 'facet' out of the name of the feature attribute because the feature's attribute is part of the feature's abstraction layer, not part of the facet system, though it is part of the glue code that bridges the two.  Probably another reason to get rid of the marker attribute...
I can rename storage feature as the use of the term 'feature' seems to be causing confusion.